lua/lspsaga/api.lua

```diff
@@ -31,11 +31,23 @@ end
 
 M.code_action_execute = function(client_id, action, ctx)
   local client = vim.lsp.get_client_by_id(client_id)
+  local is_nightly = vim.fn.has("nvim-0.8.0")
   if
     not action.edit
     and client
-    and type(client.resolved_capabilities.code_action) == "table"
-    and client.resolved_capabilities.code_action.resolveProvider
+    and
+    (
+      ( is_nightly
+        and type(client.server_capabilities.codeActionProvider) == "table"
+        and client.server_capabilities.codeActionProvider.resolveProvider
```

Comment:
for sake of readability, `client.server_capabilities` should be abbreviated to capabilities